src/exchange/domain/domain.html

```diff
@@ -9,14 +9,12 @@
             <div class="alert alert-info"
                  role="alert"
                  data-ng-if="!ctrl.loading && ctrl.search.value == null && ctrl.paginated.domains.length === 0">
-                <div>
-                    <span data-i18n-static="exchange_tab_domain_no_domains_1"></span>
-                    <button class="btn btn-link p-0"
-                            type="button"
-                            data-i18n-static="exchange_tab_domain_no_domains_2"
-                            data-ng-click="ctrl.setAction('exchange/domain/add/domain-add')"></button>
-                    <span data-i18n-static="exchange_tab_domain_no_domains_3"></span>
-                </div>
+                <span data-i18n-static="exchange_tab_domain_no_domains_1"></span>
+                <button class="btn btn-link p-0"
+                        type="button"
+                        data-i18n-static="exchange_tab_domain_no_domains_2"
+                        data-ng-click="ctrl.setAction('exchange/domain/add/domain-add')"></button>
+                <span data-i18n-static="exchange_tab_domain_no_domains_3"></span>
```

Comment:
You're already using a `data-i18n-static` attribute directly in your `<button>` element. Are you sure that your `<span>` will have inside of it the trad `exchange_tab_domain_no_domains_3` ?